spec/models/okc/okc_blotter_booking_spec.rb

```diff
@@ -1,15 +1,9 @@
 require 'rails_helper'
 
```

Comment:
since you removed the validations you needed to cut them down here as well, got it.